Monika After Story/game/script-topics.rpy

```diff
@@ -13046,3 +13046,41 @@ label mas_show_unseen:
     m 1esa "Just give me a second.{w=0.5}.{w=0.5}.{nw}"
     m 3hua "There you go!"
     return
+    
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="installed_python",category=['miscellaneous'],prompt="I installed Python.",pool=True))
+
+label installed_python:
+    m 1sub "You did?"
+    m 1hub "That's great, [player]!"
+    m 1eub "What did you download it for?{nw}"
+    $ _history_list.pop()
+    menu:
+        m 1eub "What did you download it for?{fast}"
+        "A personal project.":
+    m 1hub "How fun!"
```

Comment:
block needs to be indented to one tab (4 spaces) in front of the menu option's indent level

Reply:
I think I fixed it!